APMrover2/Rover.h

```diff
@@ -241,10 +241,6 @@ class Rover : public AP_HAL::HAL::Callbacks {
     // This is set to -1 when we need to re-read the switch
     uint8_t oldSwitchPosition;
 
-    // These are values received from the GCS if the user is using GCS joystick
-    // control and are substituted for the values coming from the RC radio
-    int16_t rc_override[8];
-
     // A flag if GCS joystick control is in use
     bool rc_override_active;
```

Comment:
This is interesting, it's actually used in a check, but it's never set to true. For now I'm leaving it alone, but it's a signal of bad handling in rover of overrides. However since the next step is to move all override handling I don't want to chase it till that has been done.